Add playbook notifications info callout (#1141)
SEKOIA-IO/documentation: docs/getting_started/notifications-Listing_Creation.md

You can filter by type of trigger or type of actions.

!!! Warning

>     Each notification is visible only for the one that configured it AND on the community it was created.

Reviewer: ```suggestion
    Each notification is exclusively visible to the user who configured it AND within the community where it was created.
```

4. Give your notification a `name`
5. Save your changes

!!! Note 

>     **Notifications concerning playbooks**

Reviewer: @mathieubellon I was thinking maybe this section belongs to [this page](https://docs.sekoia.io/getting_started/notifications-Examples/)? Or deserves a section "Playbooks' notifications"
